operations/login_operation.go

```diff
@@ -9,6 +9,7 @@ import (
 	"encoding/json"
 
 	"github.com/degica/barcelona-cli/api"
+	"github.com/degica/barcelona-cli/config"
```

Comment:
Please don't add this dependency if you don't need to. The re-login should take place inside the `config` or `client` package if thats where it is used, no need to involve the user's login operation code